libraries/chain/include/eosio/chain/controller.hpp

```diff
@@ -58,6 +58,7 @@ namespace eosio { namespace chain {
 
             genesis_state            genesis;
             wasm_interface::vm_type  wasm_runtime = chain::config::default_wasm_runtime;
+            fc::microseconds         abi_serializer_max_time_ms = chain::config::default_abi_serializer_max_time_ms;
```

Comment:
We shouldn't store this on controller as it is only slightly better than storing on abi_serializer.